_i18n/en.yml

```diff
@@ -33,24 +33,21 @@ index:
     content: |
       login.gov offers the public secure and private online access to participating government programs. With one login.gov account, users can sign in to multiple government agencies. Our goal is to make managing federal benefits, services and applications easier and more secure.
   column_1:
-    heading: Simple *and* secure for the public
+    heading: Simple and secure for the public
     content:
-      p_1: Because login.gov is a shared service,
-      li_1: Users need fewer passwords and learn fewer interfaces.
-      li_2: Security experts protect one service instead of many.
-      li_3: Dedicated teams of design and security experts will continuously improve it.
+      p_1: Users need fewer passwords and learn fewer interfaces. Security experts protect one service instead of many. Dedicated teams of design and security experts will continuously improve it.
```

Comment:
Oh, that seems odd now without the first line. Maybe just add it back in as an introductory phrase to the first sentence?